substrate/frame/src/lib.rs

```diff
@@ -569,7 +569,8 @@ pub mod deps {
 	pub use sp_core;
 	pub use sp_io;
 	pub use sp_runtime;
-
+	pub use sp_statement_store;
```

Comment:
All of the primitives of the statemnt store are really just used in one pallet, I would not add them here and keep them as a dependency.

Reply:
Yeah